_posts/pl/2023-07-24-czy-wiesz-czym-jest-indexeddb.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Czy wiesz czym jest IndexedDB?"
+date: 2023-07-24T11:00:00+01:00
+published: true
+didyouknow: true
```

Comment:
to jest trochę bardziej obszerny wpis, niż czy wiesz że. Może warto go inaczej nazwać i przestawić tę wajchę na false?